domains/prometheus0719.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "prometheus0719",
+        "email": "ibarrayair545@gmail.com"
+    },
+    "record": {
+        "CNAME": "Sistema_Tickets.github.io"
```

please remove the custom domain from your github repo, you can add it back once the PR has been merged
```suggestion
        "CNAME": "sistema_tickets.github.io"
```